scripts/check_container_name.py

```diff
@@ -1,39 +1,49 @@
 #!/usr/bin/env python
 import re
-import sys
-from EPPs.common import StepEPP
+
+from EPPs.common import StepEPP, InvalidStepError
 
 
 class CheckContainerName(StepEPP):
     """
-    Checks that the container name assigned by the user has the correct format for a seqlab plate with the standard
-    seqlab prefix "LP[0-9]{7}-" and the suffix specified by an argument.
+    Checks that the container name(s) assigned by the user has the correct format for a seqlab plate with the standard
+    seqlab prefix "LP[0-9]{7}-" and the suffix(es) specified by an argument. The number of suffixes is no limited but
+    script assumes that each suffix is only used once and that the suffixes are applied to the output containers in
+    the same order as they appear in the suffix argument
     """
-
+    _use_load_config = False  # prevent the loading of the config file
     def __init__(self, argv=None):
         # extra suffix argument required in addition to standard arg parser arguments
         super().__init__(argv)
         self.suffix = self.cmd_args.suffix
+        self.suffix = self.cmd_args.suffix
 
     @staticmethod
     def add_args(argparser):
         argparser.add_argument(
-            '-x', '--suffix', type=str, help='Set the suffix of the container name (hyphen present in prefix)'
+            '-x', '--suffix', nargs='*',
+            help='Set the suffix of the container name(s) in order plates should appear'
         )
 
     def _run(self):
         """
-        Assemble the container name template from the fixed prefix and the suffix determined by self.suffix. Find all
-        the output container names and then check that they match the template. If not, then sys.exit with a useful
-        message.
+        Check to see if the names of the containers in the step match any of the allowed name formates which are defined by the fixed name_template
+        prefix and a number of suffixes supplied by suffix argument.
         """
-        name_template = 'LP[0-9]{7}-' + self.suffix
         containers = self.process.output_containers()
 
+        suffixes = self.suffix
+
+
         for container in containers:
-            if not re.match(name_template, container.name):
-                print('%s is not a valid container name. Container names must match %s' % (container.name, name_template))
-                sys.exit(1)
+            valid_container='False'
```

Comment:
Can we use booleans for `valid_container`?

Reply:
changed to boolean